### PR TITLE
Change description and add source to metadata

### PR DESCRIPTION
**Pull request by arturo-seijas**
Metadata changes to adapt to discourse as per https://discourse.charmhub.io/t/changes-to-charmhub-io-pages/9366

### CODE REVIEW

**Review by jdkandersson — APPROVED** [review]

**Review by weiiwang01 — APPROVED** [review]

**Review by gregory-schiano — DISMISSED** [review]
LGTM

**Review by varshigupta12 — APPROVED** [review]

**Review by nrobinaubertin — APPROVED** [review]

**Review by gregnebu — APPROVED** [review]
LGTM

**Review by gregory-schiano — DISMISSED** [review]
LGTM

File: tests/integration/conftest.py [hunk 1]
```diff
@@ -225,3 +137,50 @@ async def _deploy_any_charm(src_overwrite):
...
     return _deploy_any_charm
+
+
+@pytest_asyncio.fixture(scope="module")
+async def build_and_deploy(model: Model, run_action, build_and_deploy_ingress, deploy_any_charm):
+    """build and deploy nginx-ingress-integrator charm.
+
+    Also deploy and relate an any-charm application for test purposes.
+
+    Returns: None.
```
Comment (nrobinaubertin) [inline tests/integration/conftest.py:148]: Yields the app ?

Comment (nrobinaubertin) [reply tests/integration/conftest.py:148]: Okay, it returns None. But the name `application` should probably not be used

**Review by gtrkiller — APPROVED** [review]
LGTM

**Review by gregory-schiano — APPROVED** [review]
Should be the good one :)

**Review by nrobinaubertin — APPROVED** [review]

**Pull request merged**